docker/Dockerfile-prod-opt

```diff
@@ -83,20 +84,20 @@ RUN echo "build_opt: $BUILD_OPT" > /workdir/sim_telarray/build_opts.yml && \
 FROM almalinux:9.5-minimal
 WORKDIR /workdir
 ARG BUILD_BRANCH=main
+ARG SIMTOOLS_REQUIREMENT="git+https://github.com/gammasim/simtools.git"
 COPY --from=build_image /workdir/sim_telarray/ /workdir/sim_telarray/
 
 RUN microdnf update -y && microdnf install -y \
     bc findutils gcc-c++ git gsl-devel gcc-fortran libgfortran \
-    procps python3.11-pip zstd && \
+    procps python3.12-devel zstd && \
```

Comment:
Should the python version be an argument that updates when we updated the supported versions?

Reply:
Good point, added.